### ISSUE TITLE
application unable to install since v2.16.0

### ISSUE THREAD

**Issue by Seamus-Ohhhh** [quasarframework/quasar#17190]
### What happened?

Using Quasar as an imported module the application is failing to mount due to error thrown when installing quasar. 

since the upgrade to v.2.16.0 Quasar tries to install in SSR mode. this is due to `__QUASAR_SSR_SERVER__` returning true when the application is building. the install function is expecting 3 arguments.

### What did you expect to happen?

application mounts as normal in client mode

### Reproduction URL

https://codesandbox.io/p/sandbox/vue3-base-forked-4zx2vs?file=%2Fsrc%2Fmain.js

### How to reproduce?

1. open link
2. wait for app to load (or not in this case)

### Flavour

UMD

### Areas

SPA Mode

### Platforms/Browsers

Firefox, Chrome, Safari

### Quasar info output

```Shell
Operating System        Linux(5.15.0-106-generic) - linux/x64
NodeJs                  20.13.1

Global packages    
  NPM                   10.7.0
  yarn                  1.22.22
  pnpm                  9.1.0
  bun                   Not installed
  @quasar/cli           2.4.1
  @quasar/icongenie     Not installed
  cordova               Not installed
```

### Relevant log output

```Shell
TypeError
can't convert undefined to object
Ff
/node_modules/quasar/dist/quasar.server.prod.cjs:7:19684
```

### Additional context

is there a way to manually set SSR mode to false through the config passed with app.use(Quasar)? 

can get the application to mount by passing an ssrConfig object like below but this leads to other issues within the application

`const ssrConfig = {
    req: {
        headers: '',
    }
}`

**Comment by rstoenescu** [comment]
Pushed a fix for the legacy build systems that don't know about "exports" field.
Will be available tomorrow in Quasar v2.16.4.

**Comment by Seamus-Ohhhh** [comment]
issue still showing when updating to v 2.16.4 

have updated the codesandbox to show
